quarkus/tests/integration/src/test/resources/org/keycloak/it/cli/dist/approvals/cli/help/HelpCommandDistTest.testStartDevHelp.unix.approved.txt

```diff
@@ -150,14 +150,14 @@ HTTP(S):
 --https-protocols <protocols>
                      The list of protocols to explicitly enable. Default: TLSv1.3,TLSv1.2.
 --https-trust-store-file <file>
-                     DEPRECATED: The trust store which holds the certificate information of the
-                       certificates to trust.
+                     DEPRECATED. Use: truststore-paths. -- The trust store which holds the
+                       certificate information of the certificates to trust.
 --https-trust-store-password <password>
-                     DEPRECATED: The password of the trust store file.
+                     DEPRECATED. Use: truststore-paths. -- The password of the trust store file.
```

Comment:
There will be no replacement for the password - the the store requires a password, the user will have to convert it.

Reply:
I'll add a more descriptive note.

Reply:
Change the wording a bit and removed the hinted option.